randovania/generator/filler/weights.py

```diff
@@ -1,38 +1,38 @@
 import dataclasses
 from dataclasses import dataclass
-from typing import Final
+from typing import ClassVar
 
 
 @dataclass(frozen=True)
 class ActionWeights:
-    DEFAULT_INDICES_WEIGHT: Final[float] = 1.0
+    DEFAULT_PICKUP_INDICES_WEIGHT: ClassVar[float] = 1.0
```

Comment:
btw to explain this change: `ClassVar` attributes in a dataclass are not treated as dataclass fields, so they're omitted from `__init__()` and `dataclasses.fields()`. this better reflects the intention of these attributes